lib/tests/streamlit/runtime/legacy_caching/caching_test.py

```diff
@@ -436,12 +436,12 @@ def st_warning_text_func():
             normalize_md(el.exception.message),
             normalize_md(
                 """
-Your script uses `st.markdown()` or `st.write()` to write to your Streamlit app
+Your script uses `st.markdown()` to write to your Streamlit app
 from within some cached code at `st_warning_text_func()`. This code will only be
 called when we detect a cache "miss", which can lead to unexpected results.
 
 How to fix this:
-* Move the `st.markdown()` or `st.write()` call outside `st_warning_text_func()`.
+* Move the `st.markdown()` call outside `st_warning_text_func()`.
```

Comment:
Hm, may not be worth fixing since `st.cache` is in maintenance mode at this point anyway (this is also an orthogonal change to this docs change): do we show this warning and only run a call to `st.markdown` on the first cached function invocation and not in subsequent ones? It feels weird that `st.write` calls that do eventually get rendered as markdown get replayed but directly calling `st.markdown` doesn't.